Makefile

```diff
@@ -151,7 +151,7 @@ clean:
 ifneq (,$(filter $(shell uname),Linux Darwin GNU/kFreeBSD GNU OpenBSD FreeBSD DragonFly NetBSD MSYS_NT Haiku))
 
 HOST_OS = POSIX
-CMAKE_PARAMS = -DZSTD_BUILD_CONTRIB:BOOL=ON -DZSTD_BUILD_STATIC:BOOL=ON -DZSTD_BUILD_TESTS:BOOL=ON -DZSTD_ZLIB_SUPPORT:BOOL=ON -DZSTD_LZMA_SUPPORT:BOOL=ON -DCMAKE_BUILD_TYPE=Release
+CMAKE_PARAMS = -DZSTD_BUILD_CONTRIB:BOOL=ON -DZSTD_BUILD_STATIC:BOOL=ON -DZSTD_BUILD_TESTS:BOOL=ON -DZSTD_ZLIB_SUPPORT:BOOL=ON -DZSTD_LZMA_SUPPORT:BOOL=ON -DZSTD_MULTITHREAD_SUPPORT:BOOL=ON -DCMAKE_BUILD_TYPE=Release
```

Comment:
do you means `-DZSTD_MULTITHREAD` ?

Reply:
Seems like CMake uses a different var for multithreading:
https://github.com/facebook/zstd/blob/12c045f74d922dc934c168f6e1581d72df983388/build/cmake/CMakeLists.txt#L96

Reply:
ok ! I was confused because `-DNAME` feels similar to `gcc` method to add preprocessor macro `NAME` on command line, 
but apparently it is the very same convention used by `cmake` to define its own variables,
hence the confusion.
Anyway, in such a case, the `cmake` build script can indeed use its own naming scheme.

The other thing that feels a bit unexpected here 
is to define `CMAKE_PARAMS` so far away from the only place which uses it.
It looks to me that this variable definition should be brought down closer to `cmakebuild`,
which is the only recipe which uses it.
(pay attention to the `ifneq (,$(filter MSYS%,$(shell uname)))` just above which also redefines `CMAKE_PARAMS`).